### DISCUSSION TITLE
为何同时使用 next.js 和 react router？

### DISCUSSION THREAD

**Discussion by chshouyu** [ChatGPTNextWeb/NextChat#2886] [General]
看了下源码，感觉 next.js 的存在价值不高，直接 vite + react + react router 不就好了，何必引入 next.js

> Whenever possible, we recommend fetching data on the server. 

next.js 13 目前主推服务端渲染，但其实这个项目是偏前端的

**Comment by Yidadaa** [comment]
需要用 vercel 的 edge function 做转发。
